tests/unit/records/test_base.py

```diff
@@ -330,3 +333,31 @@ def test_get_records_ids_by_pids_function_is_properly_called_with_parameters(PI_
 
     assert PI_mock.call_count == 2
     assert PI_mock.call_args_list == expected_call_list
+
+
+def test_populate_recid_from_ref():
+
+    record_data = {
+        "simple_key": {"$ref": "http://x/y/1"},
+        "key_with_record": {"$ref": "http://x/y/2"},
+        "record": {"$ref": "http://x/y/3"},
+        "embedded_list": [{"record": {"$ref": "http://x/y/4"}}],
+        "embedded_record": {"record": {"$ref": "http://x/y/5"}},
+    }
+
+    ConferencesMetadataRawFieldsSchemaV1().populate_recid_from_ref(record_data)
```

Comment:
why not create a dummy schema class and use directly the mixin on it and test it that way?

Reply:
+1

Reply:
?